Mathlib/CategoryTheory/Monoidal/Mon_.lean

```diff
@@ -412,7 +412,7 @@ theorem Mon_tensor_mul_one (M N : Mon_ C) :
     (M.X ⊗ N.X) ◁ ((λ_ (𝟙_ C)).inv ≫ (M.one ⊗ N.one)) ≫
         tensor_μ C (M.X, N.X) (M.X, N.X) ≫ (M.mul ⊗ N.mul) =
       (ρ_ (M.X ⊗ N.X)).hom := by
-  simp only [whiskerLeft_comp_assoc]
+  simp only [MonoidalCategory.whiskerLeft_comp_assoc]
```

Comment:
Although this comment is not directly related to this PR, what do you think about renaming `CategoryTheory.whiskerLeft_comp` to `CategoryTheory.NatTrans.whiskerLeft_comp` or `CategoryTheory.Functor.whiskerLeft_comp` in another PR to avoid this kind of fixes?

Reply:
If we do this name change, in principle, for consistency, we should also rename `CategoryTheory.whiskerLeft` to `CategoryTheory.NatTrans.whiskerLeft`, and similarly `isoWhiskerLeft` -> `NatIso.whiskerLeft`, etc? I am not sure we need to do the change, but I would be ok with it.